<?xml version="1.0"?>
<launch>

	<node name="robot_state_publisher" pkg="robot_state_publisher" exec="robot_state_publisher">
		<param name="robot_description" value="$(command 'xacro $(find-pkg-share neo_mp_400-2)/robot_model/mp_400/mp_400.urdf')"/>
	</node>

	<!-- start node -->
	<node pkg="pilot_ros_bridge" exec="pilot_ros_bridge_node" name="pilot_ros_bridge_node">

		<param name="pilot_node" value="$(env HOME)/pilot/.pilot_main.sock"/>

		<param name="pilot_config" value="$(find-pkg-share pilot_ros_bridge)/config/default/mp-400/"/>
	</node>

</launch>
